Generate_graph.py

```diff
@@ -69,14 +71,33 @@ def main(country_iso3):
     logger.info(f'Wrote out to {outfile}')
 
 
+def initialize_with_mobility(main_dir, country_iso3):
+    filename = os.path.join(main_dir, MOBILITY_DIR, MOBILITY_FILENAME.format(country_iso3=country_iso3))
+    logger.info(f'Reading in mobility from {filename}')
+    mobility = pd.read_csv(filename)
+    mobility.set_index("ADM", inplace=True)
+
+    # set diagonal to 1 (should this be moved into code that produces mobility matrix?)
+    for adm in mobility.index:
+        mobility[adm].loc[adm] = 1
```

Comment:
Is the diagonal not set to 1 already? This should be taken care of in the mobility matrix repo

Reply:
It wasn't when I grabbed the data. Do you want me to make a new PR without this commit?

Reply:
Ah sorry, so the latest matrix is [here](https://github.com/OCHA-DAP/pa-movement-patterns-matrix/blob/master/output/afg/afg_mobility_matrix.csv). 

No need to make a new PR I think, just make the change on this branch